python/tvm/relay/frontend/pytorch.py

```diff
@@ -3142,12 +3178,13 @@ def create_convert_map(self):
             "prim::NumToTensor": self.numtotensor,
             "prim::ImplicitTensorToNum": self.tensortonum,
             "aten::ScalarImplicit": self.tensortonum,
-            "aten::constant_pad_nd": self.make_pad("constant"),
-            "aten::reflection_pad1d": self.make_pad("reflect"),
-            "aten::reflection_pad2d": self.make_pad("reflect"),
-            "aten::replication_pad1d": self.make_pad("edge"),
-            "aten::replication_pad2d": self.make_pad("edge"),
-            "aten::replication_pad3d": self.make_pad("edge"),
+            "aten::pad": self.pad,
```

Comment:
Seems to me that only with this change `test_forward_functional_pad` was able to work on my local end, is this test disabled before on CI? This is also the reason why I am not adding a new test case. @masahi

Reply:
They should be tested. There are tests like `test_forward_constant_pad1d` and `test_forward_reflection_pad1d()` in `pytorch/test_forward.py`.

Can you test on PT 1.10? This is the version we use on CI. From our earlier conversation, I'm guessing that `aten::pad` is not in PT 1.10, so we cannot test his on CI.